addons/binding/org.openhab.binding.nikohomecontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nikohomecontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd"><!--Niko Home Control Gateway Bridge Type -->
+
+	<bridge-type id="bridge">
+		<label>Niko Home Control Bridge</label>
+		<description>This bridge represents a Niko Home Control IP-interface</description>
+		<config-description>
+			<parameter name="ADDR" type="text" required="false">
+				<label>IP or Host Name</label>
+				<description>Address will be resolved on connection if not configured</description>
+				<advanced>true</advanced>
```

Comment:
That's a very valid question, I didn't think of that. In general, it would be possible to directly access the OSGi Configuration Admin and get the configuration for the binding out of it in the DiscoveryService code - but that isn't really nice.
Actually, there are quite some other bindings that have the requirement to figure out the correct network interface under which the openHAB instance is accessible. Would it make sense to have the user configure it and make the value available through a new method in NetUtil?
Another option is to do it similarly to UPnP and mDNS: Simply do the broadcast on ALL available interfaces :-)